tests/Php72/Php72Test.php

```diff
@@ -109,8 +109,14 @@ public function testSapiWindowsVt100SupportWarnsOnInvalidInputType()
             $this->markTestSkipped('Windows only test');
         }
 
-        $this->setExpectedException('PHPUnit\Framework\Error\Warning', 'expects parameter 1 to be resource');
-        sapi_windows_vt100_support('foo', true);
+        try {
+            sapi_windows_vt100_support('foo', true);
+        } catch (\PHPUnit\Framework\Error\Warning $e) {
+            $this->expectException('PHPUnit\Framework\Error\Warning');
+            $this->expectExceptionMessage('expects parameter 1 to be resource');
+
+            throw $e;
+        }
```

Comment:
I don't quite understand these changes. Are we excepting a warning or are we not?

Reply:
We are. But we also want skips to work, and this is how.

Reply:
As long as `markTestSkipped()` is called before `expectException()`, we should be good, shouldn't we?

Reply:
As long as yes, that's not the case here :)
The test listener creates a dummy implementation of the polyfilled function when the native one is not available, that throws. It used to work seamlessly, but doesn't anymore apparently after bumping phpunit.

Reply:
That's odd. Well okay, let's leave it ad it is for now and investigate later. 🙃